examples/oneview_fc_network.yml

```diff
@@ -57,6 +57,7 @@
           linkStabilityTime: '30'
       delegate_to: localhost
 
+    # This feature is available only till Fusion 3.1
```

Comment:
Not sure if Fusion i recognized externally. So better to keep it as OneView 3.10

Reply:
Changed.